composer.json

```diff
@@ -32,5 +37,14 @@
         "Disable Custom Port": "https://raw.githubusercontent.com/pantheon-systems/wp-pantheon-decoupled/main/disable-custom-port.patch"
       }
     }
+  },
+  "scripts": {
+    "lint:php": "find ./pantheon-decoupled-example.php ./pantheon-decoupled.php ./ -name '*.php' -exec php -l {} \\;",
+    "lint:phpcs": "phpcs -s --ignore=tests/* --standard=Pantheon-WP .",
+    "lint:phpcbf": "phpcbf -s --ignore=tests/* --standard=Pantheon-WP .",
+    "lint": "composer lint:php && composer lint:phpcs",
+    "phpunit": "vendor/bin/phpunit",
+    "test": "@phpunit",
+    "test:install": "bash bin/install-wp-tests.sh wordpress_test `whoami` '' localhost latest"
```

Comment:
Totally work-around-able, but the test:install command with `whoami` doesn't work for me, while the version of this from preview that uses root does. I can either make that change when I need to, or create a new mysql user for this until we get a more scalable testing solution in place.

Nice job porting these tests by the way.